src/components/Drawer/Drawer.tsx

```diff
@@ -62,80 +57,66 @@ export interface Props {
  * import {Drawer} from "@chanzuckerberg/eds";
  * ```
  *
- * TODO: update this comment with a description of the component.
+ * A window component that slides from and out of the right side of the screen.
  */
 export const Drawer = ({
   'aria-describedby': ariaDescribedBy,
   'aria-labelledby': ariaLabelledBy,
-  className,
+  drawerContainerClassName,
   isActive,
   children,
   dismissible,
   onClose,
-  closeButtonText,
-  showBackdrop = false,
+  className,
   ...other
 }: Props) => {
   /**
    * Initialize states, constants, and refs
    */
-  const [isMounted, setIsMounted] = useState(false);
-  const BODY_DISABLED_CLASS = `eds-body-is-disabled`;
-  const [activeFocus, setActiveFocus] = useState(isActive || false);
+  const [activeFocus, setActiveFocus] = useState(isActive);
   const windowRef = useRef<HTMLElement | null>(null);
   const ref = useRef<HTMLDivElement | null>(null);
-
-  /**
-   * Get previous prop
-   * 1) This is used to compare the previous prop to the current prop
-   *
-   * TODO: improve `any` type
-   */
-  function usePrevious(isActive: any) {
-    useEffect(() => {
-      ref.current = isActive;
-    });
-    return ref.current;
-  }
+  const isMountedRef = useRef(true);
 
   /**
    * Use effect
    * 1) Set prevIsActive to previous isActive prop
    * 2) If prevIsActive is defined and previous isActive prop is not equal
    * to current isActive prop, toggle state
    */
-  const prevIsActive = usePrevious(isActive); /* 1 */
+  // const prevIsActive = usePrevious(isActive); /* 1 */
```

Comment:
@jinlee93 do you remember why you left this line in after commenting it out? Is it safe to remove this line in the codebase?

Reply:
Oop yea I actually think that entire codeblock explaining the `useEffect` can be removed as I think it's pretty self-explanatory